### ISSUE TITLE
Model Point Class generates hidden GLB file on MacOS

### ISSUE THREAD

**Issue by KirsVantas** [func-godot/func_godot_plugin#49]
I've setup a FuncGodotFGDModelPointClass for a scene I have, and added it as an entity definition to my FGD file. I have configured a Game Path Models Folder in my func_godot_local_config.tres. The directory does exist. Clicking "Export File" in the FGD file outputs that it exported the file and model with no errors. Navigating to my res folder in terminal and running "find * -name [model_name].glb" reports the existence of the model file. Opening the folder in finder, running "ls -a", and opening trenchbroom and putting the entity in my map all report that the file does not exist.
I have absolutely no idea what the issue could be, I've tried verifying the disk, creating a new folder and deleting the old one, restarting my computer, and restarting finder.
macOS 14.4
FuncGodot 2024.1
Godot v4.3.stable.mono.official

**Comment by RhapsodyInGeek** [comment]
Unfortunately I'm not terribly familiar with MacOS anymore, so I'll be of limited assistance here. If I'm reading this correctly, this is the sequence of events:
- Create a Model Point Class resource
- Export FGD, GLB file is generated
- You can find the file through the terminal command line
- You cannot find the file navigating to the folder in Finder
- TrenchBroom also cannot find the file

I have doubts that this is a FuncGodot specific issue, as the saving process of the GLB just uses the Godot GLTFDocument class' `write_to_filesystem` method. I would recommend looking around for anyone reporting issues with saving generated resources or the GLTFDocument class on Mac, or even just general MacOS file visibility issues.

**Comment by KirsVantas** [comment]
I don't think its invisible, it doesn't appear when showing hidden files is on or when running "ls -a", both of which show hidden files like .gdignore. It also can't seem to be accessed by anything other than the "find" command. Commands like "cd" can't open the removed directory and commands like "cat" can't find the file. Other exports like the FGD file itself work fine, do those use a different method?

**Comment by KirsVantas** [comment]
I just now discovered a "Users" folder in my resources folder, that had the models in it (after going down all the folders it had)! I know I had tried both the absolute and relative paths for the "Game Path Models Folder" without any results, but now after switching it to a relative path it works fine. Don't know how long this folder has existed but I guess that was the issue. It's a little confusing imo since every other path there is absolute but that may just be me.
